lib/widgets/inputItems.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/material.dart';
+import 'package:todo/helpers/colorHelper.dart';
+import 'package:todo/models/enums/inputFieldEnums.dart';
+
+class InputItems extends StatelessWidget {
+  final IconData inputIcon;
+  final String inputHint;
+  final InputFieldEnums inputType;
+  final IconData? togleIcon;
+  final bool obscureText;
+  final Function? onPressed;
+  TextEditingController controler = TextEditingController();
+
+  InputItems({
+    super.key,
+    required this.inputIcon,
+    required this.inputHint,
+    required this.inputType,
+    this.togleIcon,
```

Comment:
ako su ove promenljive vec potrebne neka praksa je da idu na pocetak konstuktora redom kojim se zahtevaju, znaci this.togleIcon pa posle this.onPressed. Ovi named parametri koji su required ne moraju da imaju odredjeni redosled kojim ih slazes u konstruktoru